tgui/packages/tgui/interfaces/common/sorting/constant.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Would be `type.ts`, this isn't a constant any more (was used within where it was extracted from).

Reply:
renamed